openedx/core/djangoapps/user_api/accounts/utils.py

```diff
@@ -37,10 +38,12 @@ def validate_social_link(platform_name, new_social_link):
     # Ensure that the new link is valid.
     if formatted_social_link is None:
         required_url_stub = settings.SOCIAL_PLATFORMS[platform_name]['url_stub']
-        raise ValueError(_(
-            ' Make sure that you are providing a valid username or a URL that contains "' +
-            required_url_stub + '". To remove the link from your edX profile, leave this field blank.'
-        ))
+        translated_error_message1 = _('Make sure that you are providing a valid '
+                                      'username or a URL that contains')
+        translated_error_message2 = _('To remove the link from your edX profile,'
+                                      ' leave this field blank.')
+        raise ValueError(' ' + translated_error_message1 + ' "' + required_url_stub +
```

Comment:
@jmbowman I was getting an error from `pylint` as:
`i18n function _() must be called with a literal string (translation-of-non-string)`

So, I changed it in this way. Let me know if this is correct or not?

Reply:
It's good that you caught this, since only the first part of that text made it into the translation files, which has apparently led to translations mentioning the need to use a URL containing a double quote character.  But I think an even better fix would be to use a placeholder in the translated literal, which will be replaced at runtime with the correct URL stub.  Please read our [docs on creating template strings for translation](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/internationalization/i18n.html#do-not-append-strings-or-interpolate-values).

This will require all the translations of this message to be updated, but that should be ok in this case since they're incorrect anyway.

Reply:
That looks good, I updated the code to match it. Let me know if it requires any other change as well!